### PR TITLE
[docker] install helm in forge docker build

### PR DESCRIPTION
**Pull request by rustielin**
## Motivation

Install `helm` in the `forge` docker image. Helm is installed in the same way as in `dev_setup.sh`: https://github.com/diem/diem/blob/main/scripts/dev_setup.sh#L25

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

Yes
## Test Plan

Codebuild

### CODE REVIEW

**Comment by rexhoffman** [comment]
Why not use the dev-setup.sh and only specify a block of tools?   If need we can split the blocks up a bit more.   Not going to block this but just pointing out we likely want to stay cohesive with our install logic.

**Comment by rustielin** [comment]
> Why not use the dev-setup.sh and only specify a block of tools?

Actually this would be cleaner! Let me try that -- thanks for the rec

File: docker/forge/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,12 @@ RUN IMAGE_TARGETS="test" ./docker/build-common.sh
 FROM debian:buster-20210511@sha256:acf7795dc91df17e10effee064bd229580a9c34213b4dba578d64768af5d8c51
 
 RUN apt-get update && apt-get install -y libssl1.1 openssh-client wget && apt-get clean && rm -r /var/lib/apt/lists/*
-RUN cd /usr/local/bin && wget "https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl" -O kubectl && chmod +x kubectl
+
+RUN mkdir /diem
+COPY rust-toolchain /diem/rust-toolchain
```
Comment (rustielin) [inline docker/forge/Dockerfile:27]: dev_setup.sh is failing without this file. can probably remove this restriction for operational tools

Comment (rustielin) [reply docker/forge/Dockerfile:27]: got around it for now

**Comment by rustielin** [comment]
Tested via codebuild: https://us-west-2.console.aws.amazon.com/codesuite/codebuild/projects/diem-forge/build/diem-forge:6060ba5c-765c-4967-ab73-945484c56686/

**Review by rexhoffman — APPROVED** [review]
your likely getting more tools than you need, but not a ton more.

File: docker/forge/Dockerfile (hunk 1 above)
Comment (rexhoffman) [reply docker/forge/Dockerfile:27]: you probably want -o for operations which installs helm.

**Review by zihaoccc — DISMISSED** [review]

File: docker/forge/Dockerfile (hunk 1 above)
Comment (rustielin) [reply docker/forge/Dockerfile:27]: ah yea I misread the options. doing `-o -b`

**Comment by rustielin** [comment]
> source the file and call individual installs. or install tools via name?

That would be pretty convenient. It would also be useful to have a command within the script that prints what packages (and at which versions) each flag installs

File: docker/forge/Dockerfile (hunk 1 above)
Comment (rustielin) [reply docker/forge/Dockerfile:27]: Failing to install shellcheck on the latest codebuild run https://us-west-2.console.aws.amazon.com/codesuite/codebuild/projects/diem-forge/build/diem-forge:46907ab4-9305-4546-ae68-eecab46dece0/. Any ideas?

```

Reading state information...
--
1405 | E: Unable to locate package xz
1406 | The command '/bin/sh -c /diem/scripts/dev_setup.sh -o -b' returned a non-zero code: 100
1407 | docker build failed. retrying in 10 secs.


```

Comment (rustielin) [reply docker/forge/Dockerfile:27]: Eh added an option `-i` that lets us choose which exact packages to install. e.g. `-i helm -i kubectl`.

**Comment by rustielin** [comment]
>  install tools via name

Ok I've gone ahead and done this. Tested via local build, since codebuild is hitting its dockerhub rate limit for the debian base image. Let me know what you think cc @rexhoffman

File: scripts/dev_setup.sh [hunk 2]
```diff
@@ -782,6 +791,14 @@ if [[ "$OPERATIONS" == "true" ]]; then
   install_allure
 fi
 
+if [[ "$INSTALL_INDIVIDUAL" == "true" ]]; then
+  for (( i=0; i < ${#INSTALL_PACKAGES[@]}; i++ ));
+  do
+    PACKAGE=${INSTALL_PACKAGES[$i]}
+    "install_${PACKAGE}"
```
Comment (rexhoffman) [inline scripts/dev_setup.sh:798]: Nice.... consistency in naming always pays off.

**Review by rexhoffman — APPROVED** [review]
I checked the dev_setup.sh to see what would run for the install of a single package and it looks to me to be these lines: https://github.com/diem/diem/blob/26b4cca8b5fb8c610fcf31356a6364261362b703/scripts/dev_setup.sh#L669-L746

We get a potentially extra apt-get update and the install of a curl package, but I think this looks really good.   In the future I'll think about dep tree in the tooling, and how to avoid unnecessary apt-get updates/curl installs

**Comment by rustielin** [comment]
/land

Thanks for the review @rexhoffman ! We can iterate on this going forward, but this will unblock forge testing for now :)

**Comment by rustielin** [comment]
/cancel

ugh the script doesn't install the tools in the path. needs quick fix

**Comment by rustielin** [comment]
Added quick `ENV PATH "$PATH:/root/bin"` to get the tools installed by `dev_setup.sh` in the path. Sorry for the review spam..

**Review by rexhoffman — COMMENTED** [review]
looks good.

ship it.

**Review by rexhoffman — APPROVED** [review]

**Comment by rustielin** [comment]
Includes a fix to make shellcheck happy. Review ping again :) @rexhoffman

**Review by zihaoccc — DISMISSED** [review]

**Pull request merged**